dropwizard-jersey/src/main/java/io/dropwizard/jersey/jackson/JacksonMessageBodyProvider.java

```diff
@@ -29,6 +31,7 @@
  * {@link JsonIgnoreType}.)
  */
 public class JacksonMessageBodyProvider extends JacksonJaxbJsonProvider {
+    private static final Logger LOGGER = LoggerFactory.getLogger(JsonProcessingExceptionMapper.class);
```

Comment:
Yes, of course, a typical Copy & Paste error. What’s the procedure now? I’m new to this. Shall I fix it and create a new pull request, or will someone manually fix it after the merge?